dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -105,6 +105,7 @@ RUN python3 -m pip install aiohttp \
                  setuptools-rust \
                  six \
                  snappi==1.27.1 \
+                 snappi-convergence==0.4.1 \
```

Comment:
https://github.com/sonic-net/sonic-mgmt/pull/18044#pullrequestreview-2806175332 should fix this issue

Reply:
Let's merge this PR to unblock for now